### PR TITLE
Adds a suicide action for hand teles.

### PR DESCRIPTION
**Pull request by DreamwalkerSisyphe**
🆑 The Dreamweaver
add: Nanotrasen would like to remind the crew about the dangers of sticking one's head through an unstable portal...
/🆑

### CODE REVIEW

File: code/game/objects/items/teleportation.dm [hunk 1]
```diff
@@ -219,3 +219,14 @@
 		if(active_portal_pairs[i] == P)
 			return DESTINATION_PORTAL
 	return FALSE
+
+/obj/item/hand_tele/suicide_act(mob/user)
+	if(iscarbon(user))
+		user.visible_message("<span class='suicide'>[user] is creating a weak portal and sticking their head through! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```
Comment (ShizCalev) [inline code/game/objects/items/teleportation.dm]: p_their

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/objects/items/teleportation.dm (hunk 1 above)
Comment (TheDreamweaver) [reply code/game/objects/items/teleportation.dm]: @ShizCalev Go back to grammar school. It looks like they're (they are) trying to commit suicide, not their trying, wtf lol.

Comment (ShizCalev) [reply code/game/objects/items/teleportation.dm]: user.visible_message("<span class='suicide'>[user] is creating a weak portal and sticking [p_their()] head through! It looks like [user.p_theyre()] trying to commit suicide!</span>")

Comment (TheDreamweaver) [reply code/game/objects/items/teleportation.dm]: But their works in that context

Comment (iskyp) [reply code/game/objects/items/teleportation.dm]: yeah but pronouns

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by MrDoomBringer** [comment]
what you wanna do is check if the user has a head, if so drop the limb. otherwise just make the user do a different suicide (like whacking themselves with the hand tele)

File: config/admins.txt [hunk 2]
```diff
@@ -1,135 +1,136 @@
-###############################################################################################
```
Comment (ninjanomnom) [inline config/admins.txt]: line endings

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: config/admins.txt (hunk 2 above)
Comment (TheDreamweaver) [reply config/admins.txt]: @ninjanomnom explain please

**Comment by TheDreamweaver** [comment]
reee, I didn't commit changes to those though

**Pull request closed without merging**